test/cri-containerd/helpers/load_cpu.go

```diff
@@ -0,0 +1,43 @@
+package main
```

Comment:
I remember @dcantah had some concerns about putting `log.go` here (context: https://github.com/microsoft/hcsshim/pull/896#pullrequestreview-544571198). should we have a place where we put the test-helper source files?

Reply:
I think for this one specifically, it looks like you mount the binary in and use it as the init process. I'd prefer (and others chime in) that we just have this test use it's own image with the binary inside. This is what I did for the job container tests: https://github.com/microsoft/hcsshim/pull/969/files

Basically I moved all of our dockerfiles (it was only Amit's timestamp test one at the time) to cri-containerd/test-images/ and placed the source and dockerfile for the image in subdirectories so you can just go to the subdir and build the image. Then just push the image up to our ACR registry and we can use it in our tests

Reply:
good point. I think it's better for clients, since the image will already have the binary.